packages/shared/src/features/opportunity/protobuf/user-candidate-preference.ts

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
This feels like a pain to manage though? 👀
Probably ok for now, but envision we'll forget to update these etc XD